android/src/test/java/com/magicleap/magicscript/scene/nodes/layouts/LayoutUtilsTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.magicleap.magicscript.scene.nodes.layouts
+
+import com.magicleap.magicscript.scene.nodes.props.Alignment
+import com.magicleap.magicscript.scene.nodes.props.Padding
+import org.amshove.kluent.shouldEqual
+import org.amshove.kluent.shouldHaveSize
+import org.junit.Test
+
+class LayoutUtilsTest {
+
+    @Test
+    fun `should return padding map with size and indices of children count`() {
+        val childCount = 10
+
+        val childrenPaddingMap = LayoutUtils.createChildrenPaddingMap(childCount, Padding(), null)
+
+        childrenPaddingMap.shouldHaveSize(childCount)
```

Comment:
Can't we use infix notation like `childrenPaddingMap shouldHaveSize childCount` ?

Reply:
yes, we can

Reply:
sorry, there is no infix for that